### PR TITLE
Feat/is brazilian cell phone

### PR DESCRIPTION
**Pull request by alanneves**
I made the is brazilian cell phone function

### CODE REVIEW

File: src/phone/validators.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/**
+ * Validates the phone number based in the brazilian phone number format
+ * You can't use a phone number with country code
+ */
+export const isBrazilianCellPhone = (arg: string | number): boolean => {
```
Comment (mickaelnn) [inline src/phone/validators.ts:5]: I Didn't like this name, the Brazilian cell phone is defined by the country code +55 before the number, not the format.
isBrazilianCellPhoneFormat isn't better?

**Review by mickaelnn — CHANGES_REQUESTED** [review]

File: src/phone/validators.ts (hunk 1 above)
Comment (alanneves) [reply src/phone/validators.ts:5]: i don't agree. what do you think @caroltrindade?

Comment (caroltrindade) [reply src/phone/validators.ts:5]: I agree...the Brazilian cell phone is defined by the format and the +55 prefix, as you only validate the format I think the function name should show this.

Comment (alanneves) [reply src/phone/validators.ts:5]: ok

**Review by mickaelnn — APPROVED** [review]
LGTM

**Review by caroltrindade — APPROVED** [review]
LGTM

**Pull request merged**